File: src/main/java/com/seguros/dto/ClienteResponseDTO.java
package com.seguros.dto;

import lombok.Data;

import java.time.LocalDate;

@Data
public class ClienteResponseDTO {
    private Long id;
    private String nombre;
    private String apellido;
    private String email;
    private String telefono;

    private String tipoIdentificacion;
    private String numeroIdentificacion;
    private LocalDate fechaNacimiento;
    private String nacionalidad;
    private String estadoCivil;
    private String sexo;
    private String lugarNacimiento;
    private Double estatura;
    private Double peso;
    private String direccion;
}

File: frontGestion/src/app/models/cliente-response.dto.ts
export interface ClienteResponseDTO {
  id: number;
  nombre: string;
  apellido: string;
  email: string;
  telefono: string;

  tipoIdentificacion: string;
  numeroIdentificacion: string;
  fechaNacimiento: string;
  nacionalidad: string;
  estadoCivil: string;
  sexo: string;
  lugarNacimiento: string;
  estatura: number;
  peso: number;
  direccion: string;
}
